### PR TITLE
User story1 - with some mini mini stylings

### PR DESCRIPTION
**Pull request by MiketheBite**

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
+import { SWRConfig } from "swr";
+
+const fetcher = (url) => fetch(url).then((response) => response.json);
```
Comment (Kheirah) [inline pages/_app.js:5]:
```suggestion
const fetcher = (url) => fetch(url).then((response) => response.json());
```
Don't forget to call json :)

File: components/NurseTeam/NurseTeam.styled.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  z-index: 1;
+`; */
+
+/* display: flex;
+width: 39px;
+height: 41px;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+flex-shrink: 0; */
```
Comment (Kheirah) [inline components/NurseTeam/NurseTeam.styled.js:57]: It's better not to leave code that you commented out. Either use it or delete it before every PR.

File: components/NurseTeam/NurseTeam.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+export default function NurseTeam() {
+  const { data, error } = useSWR("/api/nurses", fetcher);
+  const [nurses, setNurses] = useState([]);
+
+  useEffect(() => {
+    if (data) {
+      setNurses(data);
+    }
+  }, [data]);
```
Comment (Kheirah) [inline components/NurseTeam/NurseTeam.js:20]: It's considered bad practise to create a separate state which stores ALL the data from your fetch. A better way would be to store only the ids. That way you can add additional info to your separate state without duplicating the data.

**Review by Kheirah — CHANGES_REQUESTED** [review]
Good start!

Have a look at the suggested changes and make sure that your deployment works properly. The website doesn't show any nurses. Did you add the environment variable info in your project at Vercel?

Also, add a description to your PR which summarizes what your code does. Link to the user story if necessary.

File: pages/_app.js (hunk 1 above)
Comment (MiketheBite) [reply pages/_app.js]: Oh, I completely forgot to add the environment variable. I've fixed it!

File: components/NurseTeam/NurseTeam.js (hunk 3 above)
Comment (MiketheBite) [reply components/NurseTeam/NurseTeam.js]: I will write more simple this time, no state! Then in the next ones I will add states when more necessary.

**Review by Kheirah — APPROVED** [review]
Well done!

**Pull request merged**